### PR TITLE
psuedo geospatial work

### PR DESCRIPTION
**Pull request by yulgit1**

### CODE REVIEW

File: app/models/concerns/geospatial_behavior.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  extend ActiveSupport::Concern
+  include ::CurationConcerns::GenericWorkBehavior
+  include ::CurationConcerns::BasicMetadata
+  include ::BasicGeoMetadata
+
+  included do
+   #allow members to be of any ActiveFedora::Base
+   aggregates :members, predicate: Vocab::PCDMTerms.hasMember,
+                         class_name: 'ActiveFedora::Base',
+                         type_validator: type_validator
```
Comment (eliotjordan) [inline app/models/concerns/geospatial_behavior.rb]: I don't think ```aggregates``` is necessary here, as this is already defined in [Hydya::PCDM](https://github.com/projecthydra-labs/hydra-pcdm/blob/bce6accb14aba03df5210321d2e52b5008658c72/lib/hydra/pcdm/models/concerns/pcdm_behavior.rb#L5-L7).

**Comment by yulgit1** [comment]
Removed unneeded aggregates, added type and, isType?, and filters_association to works level(geospatial,image,raster,vector), added GeoVocab

todo: spec of works associations, if ok implement at collection and file level

File: app/models/concerns/external_metadata_file_behavior.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  def concerns_vector?
+    false
+  end
+  def concerns_vector_file?
+    false
+  end
+  def concerns_external_metadata_file?
+    true
+  end 
+  def vector
```
Comment (drh-stanford) [inline app/models/concerns/external_metadata_file_behavior.rb]: I think this isn't needed here...

**Comment by drh-stanford** [comment]
I don't think the prefix `concerns_` for the condition parameters is part of the pattern

File: app/models/concerns/external_metadata_file_behavior.rb (hunk 2 above)
Comment (yulgit1) [inline app/models/concerns/external_metadata_file_behavior.rb]: will remove

**Comment by yulgit1** [comment]
does it matter? it's just a variable name, as long as the symbol in the condition matches a method in the classes

**Comment by drh-stanford** [comment]
we want to follow the hydra works naming/filtering pattern, as shown here:

https://github.com/projecthydra-labs/hydra-works/blob/master/lib/hydra/works/models/concerns/work_behavior.rb#L25

**Comment by drh-stanford** [comment]
closing since PR #23 does this now

**Pull request closed without merging**